lib/pearl_web/components/button.ex

```diff
@@ -33,4 +33,111 @@ defmodule PearlWeb.Components.Button do
     </button>
     """
   end
+
+  @doc """
+  Renders a backoffice_button.
+
+  ## Examples
+
+      <.backoffice_button>Send!</.backoffice_button>
+      <.backoffice_button phx-click="go" class="ml-2">Send!</.backoffice_button>
+  """
+  attr :type, :string, default: nil
+  attr :class, :string, default: nil
+  attr :rest, :global, include: ~w(disabled form name value)
+
+  slot :inner_block, required: true
+
+  def backoffice_button(assigns) do
+    ~H"""
+    <button
+      type={@type}
+      class={[
+        "phx-submit-loading:opacity-75 rounded-lg bg-dark text-light dark:bg-light dark:text-dark hover:bg-darkShade dark:hover:bg-lightShade/95 py-2 px-3",
+        "text-sm font-semibold leading-6 transition-colors",
+        @class
+      ]}
+      {@rest}
+    >
+      {render_slot(@inner_block)}
+    </button>
+    """
+  end
+
+  @doc """
+  Renders a primary button with icon and title.
+
+  ## Examples
+
+      <.primary_button title="Continue" />
+      <.primary_button title="Next" class="w-full" />
+      <.primary_button small />
+  """
+  attr :title, :string, default: nil
+  attr :icon, :string, default: "hero-arrow-right"
+  attr :small, :boolean, default: false
+  attr :gap, :string, default: "gap-1.5"
+  attr :class, :string, default: nil
+  attr :rest, :global, include: ~w(disabled phx-click phx-disable-with phx-target)
+
+  def primary_button(assigns) do
+    ~H"""
+    <button
+      class={[
+        "flex items-center justify-center px-6 py-3",
+        if(@small, do: "w-[50px] h-[50px]", else: ["w-[139px] h-[43px]", @gap]),
```

Comment:
Maybe we can use tailwind’s size utilities instead

```suggestion
        if(@small, do: "w-13 h-13", else: ["w-27 h-9", @gap]),
```